src/gpuarray/buffer.h

```diff
@@ -550,13 +550,7 @@ GPUARRAY_PUBLIC gpucontext *gpukernel_context(gpukernel *k);
  */
 #define GA_CTX_PROP_DEVNAME  1
 
-/**
```

Comment:
This is exactly the same as GA_CTX_PROP_MAXLSIZE0.  So I want to remove this alias since it may be confused for the maximum total local size.